### DISCUSSION TITLE
Mod Install Error bgkeyring.zip does not exist or cannot be opened

### DISCUSSION THREAD

**Discussion by KnightCaptainU7** [exult/exult#587] [General]
Using the latest stable release, 1.10.1, the installation on Windows 11 had a non-fatal error:

```
Runtime error (at 18:504):

ZIP file "C:\Users\User\AppData\Local\Temp\12-4567.tmp\bgkeyring.zip" does not exist or cannot be opened.
```

Unfortunately I finished the installation successfully before checking if the file was there.

Are the official Mods supposed to be so much smaller than the Non-Exult Mods?
![image](https://github.com/user-attachments/assets/1c9c076b-91bd-491e-8692-5cd7a8a00269)

**Comment by wench** [comment]
There is something strange going on here. I installed exult on  a system that did not have either game  on it and got that error. I copied the games on and correctly setup the paths and one  of the mod downloads got a 403 error. Deselecting the third party mods installed everything correctly with no errors.
Official mods are very small

**Comment by wench** [comment]
I can see the problem in the installer so converting this to an issue. The problem is fairly minor and will only occur if trying to install mods, but no valid install of bg is found or keyring is not selected for installed.

**Comment by wench** [comment]
issue for this is #592 move any discussion there. closing this
